README.rst

```diff
@@ -100,6 +101,27 @@ Running unit tests::
 Tox will take care of building a faux installation environment, and ensure that
 the mapped import paths work correctly.
 
+Running from source on old Linux
+================================
+
+If your Linux distribution has a different version of python 3 (such as python
+3.5 in Debian 9), it is recommended to do a user-space install with
```

Comment:
Regarding the term "user-space" -- is this a term now?  As an old kernel dev -- to me user-space means something completely different.  We used to just call that "install into your home directory" back in my day.  Let me know if that should change.

To my older ears.. "user space" here is almost offensively wrong.. but maybe I am just old.

Reply:
(FWIW user-space is the memory space of user processes -- as opposed to **kernel space** which is the address space of the kernel).

Reply:
Yeah I keep reading this back and it bugs me.  I know it sounds like a tiny nit: but I would recommend you rename this to **user-dir** rather than "user-space". User-space still means something completely different and once you know what it means, misuse of the term is less than perfect, IMHO.  (Do let me know if usage has changed and I'm just old). 

 Please edit this to read "user dir"... is my recommendation.